rules/windows/exfiltration_winrar_encryption.toml

```diff
@@ -0,0 +1,38 @@
+[metadata]
+creation_date = "2020/12/04"
+maturity = "production"
+updated_date = "2020/12/04"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies use of winrar.exe to create an encrypted rar file using the -hp option."
```

Comment:
can add this for reference (recent example of turla): 
https://www.welivesecurity.com/2020/12/02/turla-crutch-keeping-back-door-open/

![image](https://user-images.githubusercontent.com/64742097/101500286-8d9d1580-396e-11eb-85a4-49ffd73e0af4.png)